packages/alert/src/Alert.tsx

```diff
@@ -62,16 +104,42 @@ const icons = {
 export default function Alert({
   actionLinkText,
   actionLinkUrl,
-  description = '',
+  delay = 5000,
```

Comment:
Increase to `6000` as per the [spec](https://zeroheight.com/3ddd0f892/p/76cbba-alert/b/70c440)